### ISSUE TITLE
Crash on playing song loaded in playlist...

### ISSUE THREAD

**Issue by doud-49** [TT-ReBORN/Georgia-ReBORN#190]
Hello @TT-ReBORN,

Congratulations 👍 for your quality work on these last releases and documentation.

I encountered a small bug, which I post to you for feedback.

Configuration : Foobar **2.1.5 (x86)** - Windows 10 - your theme from master (**Georgia-ReBORN v3.0-RC3**)

**How To reproduce :** 
- Load a playlist.
- Start a track and it's OK.
- Empty playlist (music still playing)

**The Crash error :** 
```
profile\georgia-reborn\scripts\Base\gr-main.js
Error: Spider Monkey Panel v1.6.1-mod ({04620F16-1878-47A1-8EFE-0CE0B99566CC}: Georgia-ReBORN v3.0-RC3 by TT)
focusTrack is null
File: gr-main.js
Line: 2442, Column: 36
Stack trace:
  handlePlaybackTime@gr-main.js:2442:36
  on_playback_time@gr-callbacks.js:1124:9
```

**Fix to avoid crash :**
```
const notPlayingTrackSelection = nowPlayingTrack.Path !== focusTrack.Path;
--> corrected like this on my side : 
const notPlayingTrackSelection = (focusTrack === null || focusTrack === undefined || nowPlayingTrack === null || nowPlayingTrack === undefined) ? true : (nowPlayingTrack.Path !== focusTrack.Path);
```

Good luck.

**Comment by TT-ReBORN** [comment]
Hi @doud-49, 

thanks for the report!  I'll add the fix when return from holidays.

-TT

**Comment by TT-ReBORN** [comment]
Now fixed in commit [c1cef97](https://github.com/TT-ReBORN/Georgia-ReBORN/commit/c1cef97095ff820ad67024698fc60788f8f83f86).
